_posts/2020-02-28-newpipe-0.18.6-released.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "NewPipe 0.18.6 released: Hotfix for the last hotfix"
+short: "NewPipe 0.18.6 released"
+date: 2020-02-28 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Have you fully read the last blog post and remember the [first paragraph in the Whats New]({% link _posts/2020-02-26-newpipe-0.18.5-released.md %}#what-changes) section? Do you not and you can't care less? Well, I don't care either, I have a story for both of you...
+<!-- more -->
+
+## Another release???
+
+Yes, another release. The developers told me they are deeply sorry. And they promised they won't do it again. At least I think they did, no one ever tells me anything, at least they give me bread, water and sunlight every week... anyway. They are sorry. Let me try to explain why you have to update again.
```

Comment:
I'd say that's a bit too weird, though :D

Reply:
So you arent sorry, hmm.